Dockerfile

```diff
@@ -12,13 +12,11 @@ ENV LC_CTYPE en_US.UTF-8
 # bash:    various shell scripts
 # wget:    installing lein
 # git:     ./bin/version
-# nodejs:  frontend building
+# yarn:  frontend building
 # make:    backend building
 # gettext: translations
-RUN apk add --update bash nodejs nodejs-npm git wget make gettext
 
-# yarn:    frontend dependencies
-RUN npm install -g yarn
+RUN apk add --update bash yarn git wget make gettext
```

Comment:
@AndreaGiardini maybe you should revert the switch to grabbing latests yarn available through apk (and thus omitting the need to get NPM) versus the existing way of getting the latest yarn available though NPM?

Strictly this part of this PR is not related to your initial goal of wanting to up the runner image(s) to the maintained OpenJDK repo and getting to a supported Java version.

@senior - since you just did the PR to up the builder container dependency to OpenJDK and Node 8, you’re probably the one closest to decide on what you guys prefer here.

Reply:
I don't work much with FE stuff, but @tlrobinson might have an opinion about where to get `yarn` from.

Reply:
I could revert it if necessary. I just thought it would make sense to simplify the build process by removing a build layer and void installing extra packages (like `npm`) since they are not needed).

Let me know what I should do :) In term of different `yarn` versions... it should not make difference. Maybe getting always the same version using the package manager would make the build process actually easier to reproduce, rather than installing always `latest` from npm.